test/lib/next-modes/base.ts

```diff
@@ -733,6 +755,12 @@ export class NextInstance {
   public async browser(
     ...args: Parameters<OmitFirstArgument<typeof webdriver>>
   ): Promise<Playwright> {
+    try {
+      this.throwIfUnavailble()
+    } catch (error) {
+      Error.captureStackTrace(error, this.browser)
```

Comment:
We shouldn't need this but in absence of ignore-lists the additional ceremony is still worth it.